tutorial/practicals/LFRic/building_code/2_built_ins/builtins_driver.f90

```diff
@@ -54,8 +54,8 @@ program builtins_driver
   implicit none
 
   ! Global and local mesh data types
-  type(global_mesh_type), target  :: global_mesh
-  type(global_mesh_type), pointer :: global_mesh_ptr => null()
+  type(global_mesh_base_type), target  :: global_mesh
```

Comment:
Sorry to be nitpicky, but could you please align the `type` declarations by ``::` as they were before this change? :smiley:

Reply:
Done.